internal/cosmos-scripts/package.json

```diff
@@ -26,7 +26,7 @@
     "style-loader": "0.21.0",
     "webpack": "4.15.1",
     "webpack-cli": "3.0.8",
-    "webpack-dev-server": "3.1.11",
+    "webpack-dev-server": "^3.7.2",
```

Comment:
```suggestion
    "webpack-dev-server": "3.7.2",
```

As the rest of the dependencies, we'd like to fix this version to `3.7.2`.

Reply:
Updated in cfa9f520.